TelegramBot.py

```diff
@@ -658,7 +658,7 @@ def _cmd_state_sensor(self, target_id):
         statedict = self._get_state_filtered()
         msg = ""
         for entity in statedict:
-            if re.match('^sensor.*', entity, re.IGNORECASE):
+            if re.match('^binary_senso|^sensor.*', entity, re.IGNORECASE):
```

Comment:
Here is a small typo.
```suggestion
            if re.match('^binary_sensor.*|^sensor.*', entity, re.IGNORECASE):
```